schutzbot/mockbuild.sh

```diff
@@ -37,7 +37,7 @@ fi
 
 # Relative path of the repository – used for constructing both the local and
 # remote paths below, so that they're consistent.
-REPO_PATH=osbuild-composer/${DISTRO_VERSION}/${ARCH}/${COMMIT}
+REPO_PATH=osbuild-composer/${DISTRO_VERSION}/${ARCH}/${COMMIT}/${NIGHTLY:=false}
```

Comment:
This will add either `true` or `false` to the repo path, is that something we want or is this just for testing now?

Reply:
Yeah the tests don't seem to like that... What about adding a check whether it's nightly or not to line 50 and in case `NIGHTLY == true` print a message that this is nightly run and override the repository?

Reply:
> This will add either true or false to the repo path, is that something we want or is this just for testing now?

Does it hurt anything? I don't think so. Updated the URL to be a bit more verbose.

> Yeah the tests don't seem to like that... 

b/c I forgot to update the path in the .repo file. It has nothing to do with the actual URL itself.

> What about adding a check whether it's nightly or not to line 50 and in case NIGHTLY == true print a message that this is nightly run and override the repository?

Which is exactly the same thing as the change above: override the repository URL based on which pipeline we're using for execution.